static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -15,6 +15,11 @@ to download `data.xml`and save it into the `data` subdirectory.
 
 </details>
 
+> Note: In case of systems supporting reflink, use `df` utility to see that free 
+space on the drive didn't decline by the file size that we are adding and no 
+duplication takes place. As `du` is inaccurate in modern file systems using
+reflinks.
```

Comment:
Looks great! But it should go right before the code block that includes `du -sh .`. The point is to show that even we now have a file in the working space *and* in the `.dvc/cache` they still take the same amount of space.

Reply:
Thanks, incorporated the feedback.